doc/Tutorials/Dynamic_Map.ipynb

```diff
@@ -601,6 +601,106 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xs = np.linspace(0, 2*np.pi)\n",
+    "\n",
+    "def sin(ph, f, amp):\n",
+    "    return hv.Curve((xs, np.sin(xs*f+ph)*amp))\n",
+    "\n",
+    "kdims=[hv.Dimension('phase', range=(0, np.pi)),\n",
+    "       hv.Dimension('frequency', values=[0.1, 1, 2, 5, 10]),\n",
+    "       hv.Dimension('amplitude', values=[0.5, 5, 10])]\n",
+    "\n",
+    "sine_dmap = hv.DynamicMap(sin, kdims=kdims)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we define the amplitude dimension to be overlaid and the amplitude dimension to be gridded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%%opts GridSpace [show_legend=True]\n",
+    "sine_dmap.overlay('amplitude').grid('frequency')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Fully discretize\n",
+    "\n",
+    "Now we define a function to evaluate over a space, instead of sampling it manually we can evaluate it using the groupby method. The function is a spiral with a frequency and a first and second order phase term. Then we define the dimension values for all the parameters and declare the DynamicMap."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%opts Path (linewidth=1 color=Palette('Blues'))\n",
+    "\n",
+    "def spiral_equation(f, ph, ph2):\n",
+    "    r = np.arange(0, 1, 0.005)\n",
+    "    xs, ys = (r * fn(f*np.pi*np.sin(r+ph)+ph2) for fn in (np.cos, np.sin))\n",
+    "    return hv.Path((xs, ys))\n",
+    "\n",
+    "kdims=[hv.Dimension('f', values=list(np.linspace(1, 10, 10))),\n",
+    "       hv.Dimension('ph', values=list(np.linspace(0, np.pi, 10))),\n",
+    "       hv.Dimension('ph2', values=list(np.linspace(0, np.pi, 4)))]\n",
+    "\n",
+    "spiral_dmap = hv.DynamicMap(spiral_equation, kdims=kdims)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can make use of the groupby method directly to group over the frequency and phase dimensions, which we will display as part of a GridSpace by setting the ``container_type``. This leaves second phase variable, which we assign to an NdOverlay by setting the ``group_type``:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%%opts GridSpace [xaxis=None yaxis=None] Path [bgcolor='w' xaxis=None yaxis=None]\n",
+    "spiral_dmap.groupby(['f', 'ph'], group_type=hv.NdOverlay, container_type=hv.GridSpace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
Good question, looks like Jean-Luc forgot to address his own question :smile: I think the statement is correct as is, as long as the dimension is unbounded on either end it will default to open mode.